controllers/articlesController.js

```diff
@@ -53,7 +64,88 @@ export default class PostController {
         return res
           .status(BAD_REQUEST)
           .send({ message: error.details[0].message, status: BAD_REQUEST });
-      return res.status(500).send({ message: error, status: 500 });
+      return res
+        .status(INTERNAL_SERVER_ERROR)
+        .send({ message: error, status: INTERNAL_SERVER_ERROR });
+    }
+  }
+
+  /**
+   * @description It helps the user to bookmark the article for reading it later.
+   * @param  {object} req - The request object
+   * @param  {object} res - The response object
+   * @return {object} - It returns the request response object
+   */
+
+  static async bookmark(req, res) {
+    try {
+      const { id: userId } = req.user;
+      const { slug } = req.params;
+      const user = await User.findOne({ where: { id: userId } });
+      const article = await Article.findOne({ where: { slug } });
+      if (!article || !user)
+        return res.status(NOT_FOUND).json({
+          status: NOT_FOUND,
+          message: `The article with this slug ${slug} doesn't exist`
+        });
+      const { id: articleId } = article.dataValues;
```

Comment:
Please consider referring a specific article with its `slug` instead of `articleId` as found in README.md file.